src/utils/crypto-utils.ts

```diff
@@ -0,0 +1,20 @@
+import { createDecipheriv } from "crypto"
+
+import { config } from "@config/config"
+
+const ALGORITHM = "aes-256-cbc"
```

Comment:
```suggestion
const DECRYPTION_ALGORITHM = "aes-256-cbc"
```

Reply:
changed in https://github.com/isomerpages/isomercms-backend/pull/806/commits/d1f2659caecc368efa5bca1ee9f6389486359533!